src/ja/mangagun/src/eu/kanade/tachiyomi/extension/ja/mangagun/MangaGun.kt

```diff
@@ -11,14 +10,18 @@ import eu.kanade.tachiyomi.util.asJsoup
 import okhttp3.Cookie
 import okhttp3.HttpUrl.Companion.toHttpUrl
 import okhttp3.Request
+import org.jsoup.Jsoup
 import org.jsoup.nodes.Document
 import org.jsoup.nodes.Element
 import rx.Observable
 import java.util.Calendar
 
-private const val DOMAIN = "mangagun.net"
+private const val DOMAIN = "nihonkuni.com"
 
-class MangaGun : FMReader("MangaGun", "https://$DOMAIN", "ja") {
+/**
+ * This plugin is for the website originally named MangaGun(漫画軍), which was later renamed to NihonKuni(日本国). Please do not be confused by the names.
+ */
+class MangaGun : FMReader("NihonKuni", "https://$DOMAIN", "ja") {
```

Comment:
i might as well suggest it
```suggestion
class MangaGun : FMReader("NihonKuni", "https://$DOMAIN", "ja") {
// Formerly "MangaGun(漫画軍)"
override val id = 3811800324362294701